packages/server/api/controllers/teamsController.js

```diff
@@ -61,7 +61,22 @@ const validateTeamCode = async (body) => {
   return results[0];
 };
 
+const getTeamMembers = async (params) => {
+  const { teamId } = params;
+
+  const results = await knex.raw(`
+  select \`Employees\`.first_name, \`Employees\`.last_name, \`TeamMembers\`.id from \`Employees\` 
+inner join \`TeamMembers\` on \`TeamMembers\`.employee_id = \`Employees\`.id
+where \`TeamMembers\`.team_id = ${Number(teamId)}
+  `);
```

Comment:
I used ChatGPT to propose a better way of doing this query:

```js
const getTeamMembers = async (params) => {
  const { teamId } = params;

  // Using Knex's query builder to construct the SQL query
  const results = await knex('Employees')
    .select('Employees.first_name', 'Employees.last_name', 'TeamMembers.id')
    .join('TeamMembers', 'TeamMembers.employee_id', '=', 'Employees.id')
    .where('TeamMembers.team_id', Number(teamId));  // Safely parameterized

  return {
    teamMembers: results[0],
  };
};
```

Try it out and see if it works.

Reply:
I have tried this. But it's given me some errors.